src/qt/transactiontablemodel.cpp

```diff
@@ -626,7 +626,7 @@ QVariant TransactionTableModel::headerData(int section, Qt::Orientation orientat
             case Watchonly:
                 return tr("Whether or not a watch-only address is involved in this transaction.");
             case ToAddress:
-                return tr("Destination address of transaction.");
+                return tr("User-defined intent/purpose of the transaction.");
```

Comment:
I imagine someone looking at the tooltip will be wondering *what* a label is, so just repeating it won't really help.